### ISSUE TITLE
Deactivations API: Temporary Redirect (307) Error

### ISSUE THREAD

**Issue by Saka7** [twilio/twilio-node#613]
### Issue Summary
Deactivations API throws an expection

### Steps to Reproduce
1. Follow the official documentation at [deactivated-phone-numbers](https://www.twilio.com/docs/sms/deactivated-phone-numbers)
2. Set correct `accountSid`, `authToken`
3. Run code snippet:

### Code Snippet
```node
const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
const authToken = 'your_auth_token';
const client = require('twilio')(accountSid, authToken);

client.messaging.deactivations()
                .fetch({date: new Date(Date.UTC(2020, 8, 5))})
                .then(deactivations => console.log(deactivations.redirectTo));
```

### Exception/Log
```
RestException [Error]
    at success (../node_modules/twilio/lib/base/Version.js:63:15)
    at Promise_then_fulfilled (../node_modules/q/q.js:766:44)
    at Promise_done_fulfilled (../node_modules/q/q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (../node_modules/q/q.js:1229:9)
    at Pending_become_eachMessage_task (../node_modules/q/q.js:1369:30)
    at RawTask.call (../asap/asap.js:40:19)
    at flush (../node_modules/asap/raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  status: 307,
  code: undefined,
  moreInfo: undefined,
  details: undefined
}
```

### Technical details:
* twilio-node version: 3.49.1
* node version: v12.18.4

**Comment by childish-sambino** [comment]
Similar to https://github.com/twilio/twilio-csharp/issues/534

This issue has been added to our internal backlog to be prioritized. Pull requests and +1s on the issue summary will help it move up the backlog.

**Comment by childish-sambino** [comment]
Fixed by #614
